src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -997,20 +997,18 @@ def loadTemplate(self, positionNumber, charBufferNumber = 0x01):
         else:
             raise Exception('Unknown error')
 
-    def deleteTemplate(self, positionNumber):
+    def deleteTemplate(self, positionNumber, count = 1):
```

Comment:
We should validate the argument:
N = getStorageCapacity() - positionNumber - 1 (if N is not >= 1 we reject the call)